src/nodes/Image.tsx

```diff
@@ -243,7 +243,6 @@ export default class Image extends Node {
     useResizeObserver(resizableWrapperRef, (entry) => {
       imageResized.current = true;
       sizeRef.current.width = entry.width;
-      sizeRef.current.height = entry.height;
```

Comment:
So just to give a context about the image node, at first, we didn't have the ability to resize images, they were either small or big, and even after adding the resize functionality through CSS, we needed a way to save that resized width/height into the markdown so it won't be lost, but the markdown standards for images doesn't allow to store the width/height, i think, so i had to use the [markdown-it-imsize](https://github.com/tatsy/markdown-it-imsize) plugin. So now that we have a markdown syntax that can store the width/height of images, we were good! But now that you removed this line, I did some testing, and it seems that we're not storing the width/height anymore in the markdown syntax, so any resize will be lost after refreshing, probably because the height is not provided anymore (the deleted line). You can confirm by testing, and if that's the case, we should add this line back.